src/components/calcite-switch/calcite-switch.scss

```diff
@@ -1,149 +1,146 @@
+    @apply w-6 h-3;
+  }
+  .handle {
+    @apply w-2 h-2;
+  }
 }
 
 :host([scale="m"]) {
-  --calcite-switch-track-width: 36px;
-  --calcite-switch-track-height: 20px;
-  --calcite-switch-handle-size: 18px;
+  .track {
+    @apply w-8 h-4;
+  }
+  .handle {
+    @apply w-3 h-3;
+  }
 }
 
 :host([scale="l"]) {
-  --calcite-switch-track-width: 44px;
-  --calcite-switch-track-height: 24px;
-  --calcite-switch-handle-size: 22px;
+  .track {
+    @apply w-12 h-6;
+  }
+  .handle {
+    @apply w-5 h-5;
+  }
 }
 
 :host {
-  display: inline-block;
-  position: relative;
-  width: auto;
-  cursor: pointer;
-  user-select: none;
-  vertical-align: middle;
+  @apply inline-block 
+    relative 
+    w-auto 
+    cursor-pointer 
+    select-none 
+    align-middle;
   tap-highlight-color: transparent;
 }
 
 :host([disabled]) {
-  opacity: var(--calcite-ui-opacity-disabled);
-  pointer-events: none;
-  cursor: default;
+  @apply opacity-disabled pointer-events-none cursor-default;
 }
 // focus styles
 :host {
-  @include focus-style-base();
-  width: auto;
+  @apply focus-base w-auto;
 }
 
 :host(:focus) {
-  @include focus-style-outset();
+  @apply focus-outset;
 }
 
 .track {
-  position: relative;
-  display: inline-block;
-  vertical-align: top;
-  width: var(--calcite-switch-track-width);
-  height: var(--calcite-switch-track-height);
-  background-color: var(--calcite-ui-foreground-2);
-  border-radius: 30px;
-  border: 1px solid var(--calcite-ui-border-2);
-  pointer-events: none;
-  transition: all $transition;
+  @apply relative 
+    inline-block 
+    align-top 
+    bg-foreground-2 
+    border 
+    border-solid 
+    border-color-2 
+    box-border
+    pointer-events-auto
+    transition-all
+    duration-150
+    ease-in-out
+    rounded-full;
 }
 
 .handle {
-  position: absolute;
-  display: block;
-  width: var(--calcite-switch-handle-size);
-  height: var(--calcite-switch-handle-size);
+  @apply absolute 
+    block 
+    right-auto 
+    bg-foreground-1 
+    border-2 
+    border-solid 
+    border-color-input
+    transition-all
+    duration-150
+    ease-in-out
+    rounded-full;
   top: -1px;
   left: -1px;
-  right: auto;
-  background-color: var(--calcite-ui-foreground-1);
-  border-radius: 30px;
-  border: 2px solid var(--calcite-ui-border-input);
-  pointer-events: none;
-  transition: all $transition;
 }
 
 :host(:hover),
 :host(:focus) {
   .handle {
-    border-color: var(--calcite-switch-hover-handle-border);
-    box-shadow: inset 0 0 0 1px var(--calcite-switch-hover-handle-border);
-    right: auto;
+    border-color: theme("backgroundColor.brand");
+    box-shadow: inset 0 0 0 1px theme("backgroundColor.brand");
+    @apply right-auto;
   }
 }
 
 :host([switched]) {
   .track {
-    background-color: var(--calcite-switch-switched-track-background);
-    border-color: var(--calcite-switch-switched-track-border);
+    background-color: theme("backgroundColor.brand");
+    border-color: theme("borderColor.color-brand");
```

Comment:
☝️ could these colors be included in the tailwind config which would allow you to `@apply bg-brand border-color-brand`?

Reply:
my bad, this exists, just pushed an update.